client/src-tauri/tauri.conf.json

```diff
@@ -26,11 +26,6 @@
     "bundle": {
         "active": true,
         "targets": "all",
-        "icon": [
-            "icons/32x32.png",
-            "icons/128x128.png",
-            "icons/icon.icns",
-            "icons/icon.ico"
-        ]
+        "icon": ["icons/32x32.png", "icons/128x128.png", "icons/icon.icns", "icons/icon.ico"]
```

Comment:
Please try to not make changes that aren't directly related to the features in your PR. 

These changes often lead to annoying merge conflicts.